xbmc/utils/Mime.h

```diff
@@ -41,6 +41,504 @@ class CMime
 
 private:
   static bool parseMimeType(const std::string& mimeType, std::string& type, std::string& subtype);
+};
 
-  static const std::map<std::string, std::string> m_mimetypes;
+class CMimeTypes
+{
+public:
```

Comment:
Could this be private? Classes with public members are breaking the concept of classes as instruments of encapsulation. 

Or make CMimeTypes a struct?

Reply:
Done